doc/audit/builtin_maps.rst

```diff
@@ -257,6 +257,18 @@ JavaScript engine version of the module cache, accessible by JavaScript endpoint
 
 **Value** QuickJS version as a string.
 
+``jsengine``
```

Comment:
We seem to use the term "runtime" everywhere else (doc, type names), can this perhaps be "runtime" too, or is there a specific reason it's "engine" instead?

Reply:
Most tables names are underscore separated, we do abbreviate, but we don't concatenate words afaict, so "js_runtime" would be more consistent:

![image](https://user-images.githubusercontent.com/4016369/196404903-ec227801-24ce-4ad0-8531-7348d0818a1d.png)

Reply:
Or even "js_runtime_options"